build.sbt

```diff
@@ -1081,9 +1082,6 @@ lazy val `maven-plugin` = (project in file("dev") / "maven-plugin")
       "-XX:MaxMetaspaceSize=384m",
       s"-Dlagom.version=${version.value}",
       s"-DarchetypeVersion=${version.value}",
-      s"-Dplay.version=${Dependencies.PlayVersion}",
-      s"-Dakka.version=${Dependencies.AkkaVersion}",
-      s"-Dscala.binary.version=${(scalaBinaryVersion in `api-javadsl`).value}",
```

Comment:
Passing in `scala.binary.version` here was masking a bug in the archetype before this pull request (it was not putting `scala.binary.version` in the generated project, so you had to put it on the command line every time).

It also didn't actually provide cross-building capability, because sbt-doge can't cross-build commands (see https://github.com/sbt/sbt-doge/issues/17).